src/blockchain/staking/getStakeHistory.ts

```diff
@@ -0,0 +1,57 @@
+import { Block, Log } from '@ethersproject/abstract-provider'
+import { BaseProvider } from '@ethersproject/providers'
+import { Abi } from '@xtoken/abis'
+import { Interface } from 'ethers/lib/utils'
+
+import { formatBigNumber, getContract } from '../utils'
+
+import { ITransactionHistory } from './types'
+
+const STAKING_HISTORY_BLOCK = 12838146
+
+export const getXTKStakeHistory = async (
+  provider: BaseProvider,
+  account: string,
+  // eslint-disable-next-line functional/no-return-void
+  onError?: (err: any) => void
+) => {
+  const network = await provider.getNetwork()
+  const stakingContract = getContract(
+    'xtkManagementStakingModule',
```

Comment:
Should import the constant from `@xtoken/abis` and use it instead: `XTK_MANAGEMENT_STAKING_MODULE`